docs/modules/ROOT/pages/backwards-compatibility.adoc

```diff
@@ -0,0 +1,35 @@
+= Backwards Compatibility
+
+OpenZeppelin Contracts uses semantic versioning to communicate backwards compatibility of its API and storage layout. Patch and minor updates will generally be backwards compatible, with rare exceptions as detailed below. Major updates should be assumed incompatible with previous releases. On this page, we provide details about these guarantees.
```

Comment:
```suggestion
OpenZeppelin Contracts for Cairo uses semantic versioning to communicate backwards compatibility of its API and storage layout. Patch and minor updates will generally be backwards compatible, with rare exceptions as detailed below. Major updates should be assumed incompatible with previous releases. On this page, we provide details about these guarantees.
```

For consistency. Also splitting long ideas into multiple lines helps with posting changes in PRs.

Reply:
We have one inconsistency in the Wizard page as well, where we use OpenZeppelin Cairo Contracts.

Reply:
agree on multiple lines, but unsure regarding the suggestion. It actually applies to all Contracts and we're aiming for consistency across projects, even if it's true in here we're talking about Cairo specifically

Reply:
>  but unsure regarding the suggestion. It actually applies to all Contracts and we're aiming for consistency across projects, even if it's true in here we're talking about Cairo specifically

I got your point, but IMO that would put us in a situation where an update in any library for other languages could make this statement false in our docs. We should be referring to the library in question, not generically about all potential OpenZeppelin contracts.

Reply:
that's highly unlikely, the plan is for all OZ Contracts projects to converge as much as possible, especially around API and SLA as much as possible. I don't think this is very important right now though